static/js/bootstrapper.js

```diff
@@ -135,6 +135,10 @@
               return redirectToLogin(dbInfo, err, callback);
             }
             callback(err);
+          })
+          .then(function() {
+            localDb.close();
+            remoteDb.close();
```

Comment:
This will only be executed if initial replication is required. It'd be good to also close on subsequent app starts which will need a `localDb.close();` around line 120.